src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -331,11 +331,19 @@ public function parseHashExpression()
         return $node;
     }
 
-    public function parsePostfixExpression(Node\Node $node)
```

Comment:
Because the method is public, adding an argument is a bc break. We should use a virtual argument here, and read it using func_get_arg. See similar logic in the codebase, mostly on branch *.4

Reply:
[Done!](https://github.com/symfony/symfony/pull/45795/commits/bffdcd22cabac5e105697b44e8e34c080dbe5f39)